lib/util/include/hse/util/platform.h

```diff
@@ -74,6 +74,9 @@ hse_readfile(int dirfd, const char *path, void *buf, size_t bufsz, int flags);
 void
 hse_meminfo(unsigned long *freep, unsigned long *availp, unsigned int shift);
 
+unsigned long
+platform_lowmem_scale(const unsigned long memgb);
+
```

Comment:
Use "hse_" as a prefix so that it's clear this symbol is from the hse lib...

Reply:
Why don't we just put hse in front of every function if that is the reasoning?

Reply:
We should (e.g., look at xxhash library).  But people are generally sloppy and don't care...

Reply:
xxhash is so small, not sure it is comparable.